<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>425</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>403</width>
      <height>479</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="Table">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;普通模式&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>55</height>
        </size>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="10" column="2">
        <widget class="QPushButton" name="button_3">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="button_1">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="button_8">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QPushButton" name="button_mul">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>×</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QPushButton" name="button_point">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QPushButton" name="button_p">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="button_clear">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="button_4">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="button_square">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>x^2</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="button_pow">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>a^x</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QPushButton" name="button_f">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>+/-</string>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QPushButton" name="button_e">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="button_pow_2">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>!</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="button_A">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>排列数A</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QPushButton" name="button_d">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>÷</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="button_right">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="button_sqrt">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>√￣</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QPushButton" name="button_min">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QPushButton" name="button_0">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="button_C">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>组合数C</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="button_back">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="button_5">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QPushButton" name="button_2">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="button_left">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>(</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QPushButton" name="button_6">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="button_m">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>|</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QPushButton" name="button_9">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="button_7">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
